cargo-dist/src/tasks.rs

```diff
@@ -1654,6 +1701,16 @@ impl<'pkg_graph> DistGraphBuilder<'pkg_graph> {
             }
 
             artifacts.push(fragment);
+
+            // Create the "pretend" updaters similar to the above for exezips
+            if self.inner.install_updater {
+                let filename = format!("{}-{}-update", release.app_name, variant.target);
```

Comment:
once again variant.id is for this

Reply:
actually you just shouldn't be computing this twice. one of these two should set the other.

Reply:
Fixed; swapped to something similar to `make_executable_zip_for_variant`.